scripts/kind-e2e/e2e.sh

```diff
@@ -86,6 +86,15 @@ function kind_import_images() {
         kind --name cluster${i} load docker-image submariner:local
         kind --name cluster${i} load docker-image submariner-route-agent:local
     done
+
+    if [ ! -z "${global_CIDRs[cluster1]}" ]
+    then
+        docker tag quay.io/submariner/submariner-globalnet:dev submariner-globalnet:local
+        for i in 1 2 3; do
```

Comment:
Don't think we need globalnet image in cluster1. This adds un-necessary delay if image not needed.

Reply:
Lets keep the code consistent with existing code. As you can see above, we are loading submariner images into cluster1 and the plan is to use cluster1 for some failover/redundancy tests.